blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionServer.java

```diff
@@ -99,16 +108,23 @@ public void startServer() throws InterruptedException {
 
         RouteMatcher router = new RouteMatcher();
         router.get("/v1.0", new DefaultHandler());
-        router.post("/v1.0/multitenant/experimental/metrics", new HttpMultitenantMetricsIngestionHandler(processor, timeout));
-        router.post("/v1.0/:tenantId/experimental/metrics", new HttpMetricsIngestionHandler(processor, timeout));
-        router.post("/v1.0/:tenantId/experimental/metrics/statsd", new HttpAggregatedIngestionHandler(processor, timeout));
+        router.post("/v1.0/multitenant/experimental/metrics",
+                new HttpMultitenantMetricsIngestionHandler(processor, timeout, ENABLE_PER_TENANT_METRICS));
+        router.post("/v1.0/:tenantId/experimental/metrics",
+                new HttpMetricsIngestionHandler(processor, timeout, ENABLE_PER_TENANT_METRICS));
+        router.post("/v1.0/:tenantId/experimental/metrics/statsd",
+                new HttpAggregatedIngestionHandler(processor, timeout, ENABLE_PER_TENANT_METRICS));
```

Comment:
Wouldnt it be simple to just get the property value inside each handler instead of changing the constructor for each handler?